jetty-websocket/websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/CloseStatus.java

```diff
@@ -343,6 +331,8 @@ public static String codeString(int closeStatus)
                 return "PROTOCOL";
             case BAD_DATA:
                 return "BAD_DATA";
+            case RESERVED:
```

Comment:
Rather than have this switch here, that has to be maintained, it really feels like we should have an enum with both the code and the string.... hmmmm

Reply:
It's difficult because we aren't restricted to these defined values here, it could be any int within the 1000-5000 range.